lib/pbench/test/unit/server/query_apis/test_datasets_modify.py

```diff
@@ -277,3 +293,147 @@ def test_invalid_owner_params(
             response.json["message"]
             == "Value 'invalid_owner' (str) cannot be parsed as a username"
         )
+
+    @pytest.mark.parametrize("owner", ("drb", "test"))
```

Comment:
This looks like you just combined the two unit test modules.

First, you don't have to do that, because there's no rule that each endpoint can only be tested by a single pytest file. It's fine to have a file for delete and a file for "modify" if that's convenient. However, you could also combine the two *test *classes** within the same file, and still have them operate independently so you don't have to worry about defining two `test_partial` methods and such on the same class...